gpii/node_modules/canopyMatchMaker/configs/standard.logging.json

```diff
@@ -0,0 +1,22 @@
+{
+    "typeName": "canopyMatchMaker.development",
```

Comment:
This config doesn't follow our new practices for naming. The filename should agree with the typeName, which should be a fully qualified namespace beginning with, e.g. gpii.matchMaker.canopy.something